README.md

```diff
@@ -1,55 +1,63 @@
-# <a href='https://www.meteor.com'><img src='https://user-images.githubusercontent.com/841294/26841702-0902bbee-4af3-11e7-9805-0618da66a246.png' height='60' alt='Meteor'></a>
+<div align="center">
+  <a href="https://www.meteor.com" target="_blank">
+    <img align="center" width="225" src="https://user-images.githubusercontent.com/841294/26841702-0902bbee-4af3-11e7-9805-0618da66a246.png">
+  </a>
+</div>
+
+<br>
+
+<div align="center">
 
 [![TravisCI Status](https://travis-ci.org/meteor/meteor.svg?branch=devel)](https://travis-ci.org/meteor/meteor)
 [![CircleCI Status](https://circleci.com/gh/meteor/meteor/tree/devel.svg?style=shield&circle-token=c2d3c041506bd493ef3795ffa4448684cfce97b8)](https://circleci.com/gh/meteor/meteor/tree/devel)
+  
+</div>
+
+<hr>
 
-Meteor is an ultra-simple environment for building modern web
-applications.
+Meteor is an **ultra-simple** environment for building **modern** web applications.
 
-With Meteor you write apps:
+<hr>
 
-* in modern JavaScript
-* that send data over the wire, rather than HTML
-* using your choice of popular open-source libraries
+- [Official Website](https://www.meteor.com)
+- [Installation](https://www.meteor.com/developers/install)
+- [Documentation](https://docs.meteor.com/#/full/)
 
-Try a getting started tutorial:
- * [React](https://react-tutorial.meteor.com)
- * [Blaze](https://blaze-tutorial.meteor.com/)
- * [Vue](https://vue-tutorial.meteor.com/)
- * [Svelte](https://svelte-tutorial.meteor.com/)
+<hr>
 
-Next, read the [documentation](https://docs.meteor.com/).
+### 📚 **Create your applications using modern JavaScript**
 
-Are you looking for examples? Check this [meteor/examples](https://github.com/meteor/examples).
+Benefit from the latest technology updates to fastly prototype and develop your applications.
 
-Check your [changes](https://docs.meteor.com/changelog.html) to keep your app up-to-date.
+<hr>
 
-## Quick Start
+### ✨ **Integrate technologies you already use**
 
-On Linux/macOS/Windows, use this line:
+Use popular frameworks and tools right out-of-the-box. Focus on building features instead of configuring disparate components yourself.
 
-```bash
-npm install -g meteor
-```
+<hr>
 
-Visit the official [install page](https://www.meteor.com/developers/install) to learn more.
+### 💻 **Build apps for any device**
 
-Create a project:
+Use the same code whether you’re developing for web, iOS, Android, or desktop for a seamless update experience for your users.
 
-```bash
-meteor create my-app
-```
+<hr>
 
-Run it:
+# 🔥 Getting Started
 
-```bash
-cd my-app
-meteor
-```
+What about to try a getting started tutorial in your favorite technology?
```

Comment:
I would keep this part of creating and running it 


## Quick Start

On your platform, use this line:

```sh
> npm install -g meteor
```

🚀 To create a project:

```sh
> meteor create my-app
```

☄️ Run it:

```sh
> cd my-app
> meteor
```

Reply:
Solved in 2c5bbb05249d3f2d68be097a340328bc521d4620